tests/e2e/controlplane/control_plane_test.go

```diff
@@ -305,9 +305,6 @@ spec:
 			By("Cleaning up the IstioCNI namespace")
 			Expect(k.DeleteNamespace(istioCniNamespace)).To(Succeed(), "IstioCNI Namespace failed to be deleted")
 
-			By("Deleting any left-over Istio and IstioRevision resources")
-			Expect(forceDeleteIstioResources()).To(Succeed())
```

Comment:
For people reviewing this PR:

The forceDeleteIstioResources API can mask any errors and is
not really required as we delete the Istio/IstioCNI resource
in the previous steps.